website/docs/cloud-docs/integrations/kubernetes/api-reference.mdx

```diff
@@ -110,7 +127,12 @@ _Appears in:_
 
 #### AgentToken
 
-HCP Terraform agents use secret tokens, called `AgentToken`s, to connect to the HCP Terraform agent pool. In `spec` only the field `Name` is allowed, the rest are used in `status`. [More information](/terraform/cloud-docs/agents)
+
+
+Agent Token is a secret token that a HCP Terraform Agent is used to connect to the HCP Terraform Agent Pool.
+In `spec` only the field `Name` is allowed, the rest are used in `status`.
+More infromation:
+  - https://developer.hashicorp.com/terraform/cloud-docs/agents
```

Comment:
i think the previous wording was a lil bit more clear. i also updated the second sentence to be active voice and fixed typo

```suggestion
HCP Terraform agents use secret tokens, called `AgentToken`s, to connect to the HCP Terraform agent pool. In `spec`, you can only define `Name`. The other attributes are returned in `status`. 

More information:
  - https://developer.hashicorp.com/terraform/cloud-docs/agents
```